apps/web/pages/[user]/book.tsx

```diff
@@ -196,6 +196,8 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
       booking,
       trpcState: ssr.dehydrate(),
       isDynamicGroupBooking,
+      hasHashedBookingLink: false,
+      hashedLink: undefined,
```

Comment:
To confirm the possible associated hashed link shouldn't be updated after the event is booked